src/main/java/org/spongepowered/common/data/manipulator/immutable/ImmutableSpongeMobSpawnerData.java

```diff
@@ -189,39 +188,39 @@ public ImmutableNextEntityToSpawnValue nextEntityToSpawn() {
         return this.toSpawnValue;
     }
 
-    public short getRemaining() {
+    private short getRemaining() {
         return this.remaining;
     }
 
-    public short getMinSpawnDelay() {
+    private short getMinSpawnDelay() {
         return this.minSpawnDelay;
     }
 
-    public short getMaxSpawnDelay() {
+    private short getMaxSpawnDelay() {
         return this.maxSpawnDelay;
     }
 
-    public short getCount() {
+    private short getCount() {
         return this.count;
     }
 
-    public short getMaxNearby() {
+    private short getMaxNearby() {
         return this.maxNearby;
     }
 
-    public short getPlayerRange() {
+    private short getPlayerRange() {
         return this.playerRange;
     }
 
-    public short getSpawnRange() {
+    private short getSpawnRange() {
         return this.spawnRange;
     }
 
-    public WeightedSerializableObject<EntitySnapshot> getNextToSpawn() {
+    private WeightedSerializableObject<EntitySnapshot> getNextToSpawn() {
         return this.nextToSpawn;
     }
 
-    public WeightedTable<EntitySnapshot> getEntitiesToSpawn() {
+    private WeightedTable<EntitySnapshot> getEntitiesToSpawn() {
         return this.entitiesToSpawn;
```

Comment:
Note that this is leaking a mutable collection, so at least, this could be duplicated on get.